src/wp-admin/includes/upgrade.php

```diff
@@ -2649,6 +2649,32 @@ function drop_index( $table, $index ) {
 	return true;
 }
 
+/**
+ * Checks if an index exists for a specified column in a table.
+ *
+ * @since 6.8.0
+ *
+ * @global wpdb $wpdb WordPress database abstraction object.
+ *
+ * @param string $table  The name of the table.
+ * @param string $column The name of the column to check for an index.
+ * @return bool True if the index exists, false otherwise.
+ */
+function has_index( $table, $column ) {
+	global $wpdb;
+
+	// Prepare the query to check if an index exists on the specified column.
```

Comment:
Imho this comment could be removed as the function is straight to the point and this comment does not add anything that wasn't obvious.

Reply:
Fixed